internal/presenters/presenter_local_finding.go

```diff
@@ -171,69 +146,40 @@ func (p *LocalFindingPresenter) RenderTemplate(templateFiles []string, mimeType
 		return err
 	}
 
-	sum := PrepareSummary(&p.Input.Summary, p.OrgName, p.TestPath, p.SeverityMinLevel)
+	summary := PrepareSummary(&p.Input.Summary, orgName, p.TestPath, severityMinLevel)
 
 	mainTmpl := localFindingsTemplate.Lookup("main")
 	if mainTmpl == nil {
 		return fmt.Errorf("the template must contain a 'main'")
 	}
 
-	filteredFindings := filterOutIgnoredFindings(p.Input.Findings, p.ShowIgnored, []string{"low", "medium", "high"})
+	openFindings, ignoredFindings := filterFindingsByIgnored(p.Input.Findings, shouldShowIgnored)
 
 	err = mainTmpl.Execute(p.writer, struct {
 		Summary         SummaryData `json:"summary"`
-		OpenFindings    []*PresentationFindingResource
-		IgnoredFindings []*PresentationFindingResource
-		ShowIgnored     bool
-		ShowDivider     bool
-		SeverityFilter  string
+		OpenFindings    []local_models.FindingResource
+		IgnoredFindings []local_models.FindingResource
 	}{
-		Summary:         sum,
-		OpenFindings:    filteredFindings.OpenFindings,
-		IgnoredFindings: filteredFindings.IgnoredFindings,
-		ShowIgnored:     p.ShowIgnored,
-		ShowDivider:     shouldShowDivider(filteredFindings, p.ShowIgnored),
-		SeverityFilter:  p.SeverityMinLevel,
+		Summary:         summary,
+		OpenFindings:    openFindings,
+		IgnoredFindings: ignoredFindings,
 	})
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
-func shouldShowDivider(findings FilteredFindings, showIgnored bool) bool {
-	hasFindings := len(findings.OpenFindings) > 0 && len(findings.IgnoredFindings) > 0
-	return hasFindings || showIgnored
-}
-
-func sortFindings(findings []*PresentationFindingResource, order []string) []*PresentationFindingResource {
-	result := make([]*PresentationFindingResource, 0, len(findings))
-
-	result = append(result, findings...)
-
-	slices.SortFunc(result, func(a, b *PresentationFindingResource) int {
-		if a.Attributes.Rating.Severity.Value != b.Attributes.Rating.Severity.Value {
-			return slices.Index(order, string(a.Attributes.Rating.Severity.Value)) - slices.Index(order, string(b.Attributes.Rating.Severity.Value))
-		}
-
-		return 0
-	})
-
-	return result
-}
-
-func filterOutIgnoredFindings(findings []local_models.FindingResource, showIgnored bool, sortOrder []string) (filtered FilteredFindings) {
+func filterFindingsByIgnored(findings []local_models.FindingResource, showIgnored bool) (openFindings []local_models.FindingResource, ignoredFindings []local_models.FindingResource) {
```

Comment:
**issue:** refactor to make this generic

Reply:
Currently tied to FindingResource type. Given a FindingsList is a primitive in our domain this seems reasonable for the time being.